internal/painter/gl/gl_common.go

```diff
@@ -120,8 +120,16 @@ func (p *glPainter) newGlRasterTexture(obj fyne.CanvasObject) Texture {
 func (p *glPainter) newGlLinearGradientTexture(obj fyne.CanvasObject) Texture {
 	gradient := obj.(*canvas.LinearGradient)
 
-	width := p.textureScale(gradient.Size().Width)
-	height := p.textureScale(gradient.Size().Height)
+	w := gradient.Size().Width
+	h := gradient.Size().Height
+	switch gradient.Angle {
+	case 90, 70:
```

Comment:
Why 70 degrees? It might just be that I’m missing but I want to know 🙂

Reply:
Ha! very good catch! Obviously, my stubby fingers removed the `2`.
No test finds this because no one uses this particular angle.